pyproject.toml

```diff
@@ -138,8 +138,6 @@ types-termcolor = { version = "^1.1.2", optional = true }
 types-psutil = { version = "^5.8.13", optional = true }
 # test framework dependencies
 docker-compose = { version = "^1.28", optional = true }
-# spell checker dependencies
-pyspelling = { version = ">=2.8.2", optional = true }
```

Comment:
wouldn't this need to contain `typos` now instead?

Reply:
No, because it isn't a pip package. You have to install it as a binary locally, and we use the GH Actions version in the CI.